src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml

```diff
@@ -17,31 +17,31 @@ limitations under the License.
 <configurator>
   <domain name="CHIP" />
 
-  <struct name="GroupKey">
+  <struct name="GroupKeyMapStruct">
     <cluster code="0x003F"/>
-    <item fieldId="0" name="fabricIndex" type="fabric_idx"/>
-    <item fieldId="1" name="groupId" type="INT16U"/>
-    <item fieldId="2" name="groupKeySetID" type="INT16U"/>
+    <item fieldId="0" name="FabricIndex" type="fabric_idx"/>
+    <item fieldId="1" name="GroupId" type="group_id"/>
+    <item fieldId="2" name="GroupKeySetID" type="INT16U"/>
   </struct>
 
-  <struct name="GroupInfo">
+  <struct name="GroupInfoMapStruct">
     <cluster code="0x003F"/>
-    <item fieldId="0" name="fabricIndex" type="INT16U"/>
-    <item fieldId="1" name="groupId" type="INT16U"/>
-    <item fieldId="2" name="endpoints" type="INT16U" array="true"/>
-    <item fieldId="3" name="groupName" type="CHAR_STRING" length="16" />
+    <item fieldId="0" name="FabricIndex" type="fabric_idx"/>
+    <item fieldId="1" name="GroupId" type="group_id"/>
+    <item fieldId="2" name="Endpoints" type="endpoint_no" array="true"/>
+    <item fieldId="3" name="GroupName" type="CHAR_STRING" length="16" />
```

Comment:
And this field should be marked optional.

Reply:
Fixed.